### DISCUSSION TITLE
Live profiling / continuous reporting

### DISCUSSION THREAD

**Discussion by AdityaGudimella** [plasma-umass/scalene#403] [Q&A]
Currently when you profile a program with Scalene, you have to wait until the program finishes running to see the output of Scalene. Is it possible to view the results as they update live in a web page?

**Comment by emeryberger** [comment]
Good question! Right now, you can invoke `--profile-interval`, which outputs profiles every so many seconds (either files or the text UI). You can refresh Scalene to get the updated JSON file, but currently this is manual. We could have the web browser try to open the same tab, which would achieve your goals, but would reset all sorting, etc. Would that be satisfactory?
